### PR TITLE
Add Justice Hemming to board members

### PR DESCRIPTION
**Pull request by Jhemming93**
## Have you run the linting command in the `README.md`?

-   [ ] yes! `npm run lint-check`

## Have you taken a look at our [contributing guidelines](https://github.com/devedmonton/DES-Website/blob/main/.github/CONTRIBUTING.md)?

-   [ ] yes!

## My node version matches the one suggested when running `nvm use`?

-   [ ] yes!

### CODE REVIEW

File: pages/board.vue [hunk 1]
```diff
@@ -155,6 +155,18 @@ export default {
...
                     slackAccount: "Salma Ibrahim",
                 },
+                {
+                    id: 8,
+                    name: "Justice Hemming",
+                    title: "Member At Large",
+                    imageLink: require("~/assets/boardPictures/justiceHemming.jpg"),
+                    description:
+                        "Justice Hemming is an emerging Junior Web Developer, completing his Web Design and Development in the Digital Media and IT program at NAIT. He has been active in the community by running events for the Junior Devs group, and looks to continue to help build a community that everyone feels welcome in by getting involved with DES.<br /><br />When it comes to building websites, Justice is more of a jack of all trades focusing on both front and back end development, he loves the aspect of how coding is ever changing with three always being more to learn.<br /><br />When Justice isn’t coding you will still find Justice probably not far from the computer, gaming with friends, or taking on his passion for cooking and baking. His favorite thing to bake is mini cherry cheesecake.",
```
Comment (MandyMeindersma) [inline pages/board.vue:164]: This sentence doesn't make sense I think?
>Justice Hemming is an emerging Junior Web Developer, completing his Web Design and Development in the Digital Media and IT program at NAIT.

I thought you completed?
Or are you saying like "completing his degree in 2023 he then went on to do xyz? or something?

Comment (MandyMeindersma) [inline pages/board.vue:164]: >ever changing with three always being more to learn.

change three to there

Comment (MandyMeindersma) [inline pages/board.vue:164]: > front and back end development

Front End and Back End development

**Review by MandyMeindersma — CHANGES_REQUESTED** [review]

File: pages/board.vue (hunk 1 above)
Comment (Jhemming93) [reply pages/board.vue:164]: I promise english is my first language

**Comment by MandyMeindersma** [comment]
Also FYI I just merged in a pr that changed the node version which means you will merge in master and then everything will be a little adventurous. Good Luck lol

**Review by MandyMeindersma — APPROVED** [review]

**Pull request merged**